### DISCUSSION TITLE
Table filters, etc.

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2300] [Office hours] [answered]
Hi! Three questions about the new table filter/misc table stuff.

1. Is there a way to adjust the width of the filter, or is it intentionally locked? I'm trying to avoid this:
![Screen Shot 2022-03-16 at 3 55 25 PM](https://user-images.githubusercontent.com/87141551/158697761-6cbc8d50-406a-4d27-8531-c6277ecb529e.png)
 
2. What does the filter area look like when filters are applied?

3. We should be using these versions of status icons in tables, correct? 
![Screen Shot 2022-03-16 at 3 57 02 PM](https://user-images.githubusercontent.com/87141551/158697988-0ee29108-8fbb-4e95-b1b3-17aebfed8cfd.png)

And not the ones that have an icon embedded in them:
![Screen Shot 2022-03-16 at 3 57 37 PM](https://user-images.githubusercontent.com/87141551/158698048-bb36543f-87f4-4ad4-8a02-a4f3042d46f1.png)

**Comment by shleewhite** [comment, marked as answer]
Hi @ceyerkes, thank you for coming to office hours! Just to follow up, we talked about:
1. You probably need to detach the filter group to change the width
2. When the filter is changed, the only thing that updates are that the apply button and clear all buttons become enabled. We are currently adding [interactive examples to the filter group pattern page](https://github.com/twilio-labs/paste/pull/2311), but they have not merged yet.
3. That is correct! We use the versions with icons embedded for direct feedback to the user, like a toast or alert. These are considered [process statuses](https://paste.twilio.design/patterns/status#process-status) because it is something the user is waiting to happen.

**Reply by ceyerkes** [reply]
Thanks Lee!
